sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/UploadBufferPool.java

```diff
@@ -137,27 +119,7 @@ buffer we will write to on the next call to write().
     and calling write. Hence there is only one worker calling getBuffer at any time.
      */
     private BufferAggregator getBuffer() {
```

Comment:
I think we can remove that method altogether and just call ctor where we need to.